src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -697,6 +697,7 @@ components:
           $ref: '#/components/schemas/PrisonWorkAndEducationRequest'
       required:
         - hopingToGetWork
+        - prisonId
```

Comment:
This is the only breaking change to the API, however I've checked the CIAG UI codebase and it's populated anyway.